action.yml

```diff
@@ -24,8 +24,12 @@ inputs:
   package-name:
     description: >
       Name of the package containing the version to delete.
-      Required if dynamically deleting oldest versions.
-    required: false
+    required: true
+
+  package-type:
+    description: >
+      Type of the package containing the version to delete.
```

Comment:
nit: the language here can be clearer since this is a customer facing help string for the param

Reply:
Changed to 👇 (https://github.com/actions/delete-package-versions/pull/83/commits/2de4236b174fbc7e7774f69ed9e0070c7cbe2281)

_Type of package. Can be one of container, maven, npm, nuget, or rubygems._